quickwit/quickwit-common/src/uri.rs

```diff
@@ -107,26 +107,6 @@ impl FromStr for Protocol {
 
 const PROTOCOL_SEPARATOR: &str = "://";
 
-#[derive(Debug, Eq, PartialEq)]
-pub enum Extension {
```

Comment:
It's not entirely removed. The concept is now a `ConfigFormat` and moved to quickwit-config.